GaelAlburo/final-project
# Feat(users)/create edit profile page

**Summary**
This PR introduces an Edit Profile page, allowing users to update their profile information. Additionally, it includes a fix for dynamically configuring a button in the services component.

**Changes**
Edit Profile Page (feat(users): create_edit_profile_page):
  Developed a fully functional Edit Profile page with form validation and input fields for user data.
  Integrated API calls to update user information.

Dynamic Button Configuration Fix (fix(services)/dinamic_config_button):
  Fixed an issue where the button configuration was not updating dynamically.
  Refactored the button logic to adapt based on the provided configuration data.

**How to Test**

Edit Profile Page:
  Navigate to the /profile route.
  Update user details and verify that changes are reflected in the profile.
  Ensure form validations and error messages function correctly.

**Notes**
The Edit Profile page lays the groundwork for enhanced user customization features.
Ensure proper API connectivity for testing user updates
